CMSgov/design-system
# [WNMGDS-1736] updated doc site footer

## Summary

Simplifying the footer on the doc site to match new designs.

## How to test

`yarn start:gatsby` to see the changes in action
<img width="1791" alt="Screen Shot 2022-07-18 at 10 19 57 AM" src="https://user-images.githubusercontent.com/33579665/179557003-0bae1fe0-5cd8-4c4b-b9e7-9edfb3cabd62.png">
